apps/framework-docs/src/pages/building/flows/setup.mdx

```diff
@@ -105,29 +118,37 @@ import { UserActivity } from "/path/to/UserActivity";
 import { ParsedActivity } from "/path/to/ParsedActivity";
 
 // The 'convertUtc' function transforms UserActivity data to ParsedActivity format.
-export default function convertUtc(event: UserActivity): ParsedActivity {
+export default function convertUtc(
+  source: UserActivity,
+): ParsedActivity[] | ParsedActivity | null {
   // Convert local timestamp to UTC and return new ParsedActivity object.
   return {
-    id: event.id, // Retain original event ID.
-    userId: "puid" + event.userId, // Example: Prefix user ID.
-    activity: event.activity, // Copy activity unchanged.
-    timestamp: new Date(event.timestamp.toUTCString()), // Convert timestamp to UTC.
+    id: source.id, // Retain original event ID.
+    userId: "puid" + source.userId, // Example: Prefix user ID.
+    activity: source.activity, // Copy activity unchanged.
+    timestamp: new Date(source.timestamp), // Convert timestamp to UTC.
   };
 }
 ```
 
-Adjust function specifics according to your data transformation needs. This structured approach ensures clarity in your
-data processing tasks, enabling Moose to efficiently handle and automate data flows.
+Adjust function specifics according to your data transformation needs. The
+function's return type can be null (meaning no transformation will take place),
+a single object, or an array of objects. This structured approach ensures
```

Comment:
Main doc change is this paragraph explaining the different return types, and the example on line 123 above. Rest are formatting changes.